fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/wsrpc/mappers/Types.kt

```diff
@@ -62,10 +54,14 @@ class POJOCollectionMapper<T>(val classRef: Class<T>) : NullableMapper<List<T>>(
 class POJOMapper<T>(val classRef: Class<T>) : NullableMapper<T>() {
 
     override fun mapNullable(rpcResponse: RpcResponse, jsonMapper: Gson): T? {
-        val raw = rpcResponse.result as? Map<*, *> ?: return null
-
-        val tree = jsonMapper.toJsonTree(raw)
+        return when {
+            rpcResponse.result is Map<*, *> -> {
+                val raw = rpcResponse.result as? Map<*, *> ?: return null
+                val tree = jsonMapper.toJsonTree(raw)
```

Comment:
Тут сработает smart cast, нет смысла от `as? Map<*, *> ?: return null`

```suggestion
                val tree = jsonMapper.toJsonTree(rpcResponse.result)
```